apps/coordinator/src/components/BCUR2/BCUR2Display.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useState, useEffect, useCallback } from "react";
+import { BCUREncoder2 } from "@caravan/wallets";
+import {
+  Box,
+  Button,
+  Paper,
+  Typography,
+  IconButton,
+  LinearProgress,
+  FormHelperText,
+} from "@mui/material";
+import {
+  PlayArrow,
+  Pause,
+  SkipNext,
+  SkipPrevious,
+  Stop,
+} from "@mui/icons-material";
+// @ts-ignore - qrcode.react v1.0.0 doesn't have proper types
```

Comment:
I wonder if we should take this as an opportunity to use a newer/more modern library?